test/python/notebooks/test_jupyter.ipynb

```diff
@@ -0,0 +1,259 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2018-10-29T17:38:20.201993Z",
+     "start_time": "2018-10-29T17:38:19.154330Z"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import time\n",
+    "cwd = os.getcwd()\n",
+    "qiskit_dir = os.path.abspath(os.path.join(cwd,\"../../..\"))\n",
```

Comment:
Can we change this to:

qiskit_dir = os.path.abspath(os.path.dirname(os.path.dirname(os.path.dirname(cwd))))

just in case running on a non-posix fs.

Reply:
Good idea.